src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -417,7 +418,7 @@ export function registerTerminalActions() {
 					{
 						primary: KeyMod.Shift | KeyCode.Tab,
 						weight: KeybindingWeight.WorkbenchContrib,
-						when: ContextKeyExpr.and(CONTEXT_ACCESSIBILITY_MODE_ENABLED, TerminalContextKeys.focus)
+						when: ContextKeyExpr.and(CONTEXT_ACCESSIBILITY_MODE_ENABLED, TerminalContextKeys.focus, ContextKeyExpr.or(terminalTabFocusContextKey, editorTabFocusContextKey))
```

Comment:
Do we need to context key at all? If tab focus mode naturally focuses the previous element, it should be the accessible buffer which will then fire the callback to get the buffer elements?

Reply:
We could just not have a keybinding by default, which would mean it would work by natural tab order when tab focus mode is on, and when it's off they could add a keybinding if they wanted it.

Reply:
I removed the keybinding and tabbing does not focus it. Probably because of this hack that we need 

https://github.com/microsoft/vscode/blob/7043f2144fe9a36374f01dfb70c30cbf94ed2605/src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts#L864-L867

Reply:
an advantage of this approach is we could remove the when clause for accessibility mode enabled as a user would have to have tab focus mode enabled for it to work, so they wouldn't find themselves in the mode unexpectedly / it would improve discoverability

Reply:
for example, no screen reader mode on and I have `shift+tab` doing something in `oh-my-zsh`, then I toggle tab moves focus on and discover/ focus the accessible buffer

https://user-images.githubusercontent.com/29464607/221907298-c4a504d4-65f0-4fbf-8d99-9d913eedb8e3.mov

Reply:
just made it so that only in accessibility mode will it apply in the editor

` ContextKeyExpr.or(terminalTabFocusContextKey, ContextKeyExpr.and(CONTEXT_ACCESSIBILITY_MODE_ENABLED, editorTabFocusContextKey))`